examples/backlist_both_tokens.py

```diff
@@ -0,0 +1,48 @@
+from flask_jwt_extended import decode_token
+
```

Comment:
Could we add the boiler plate code here, to make this a fully runnable example? Ie creating the `app` object and configuring the `flask_jwt_extended` extension?